src/mode/modeHandler.ts

```diff
@@ -484,6 +502,8 @@ export class ModeHandler implements vscode.Disposable {
         this._vimState.settings.useSolidBlockCursor = vscode.workspace.getConfiguration("vim")
             .get("useSolidBlockCursor", false);
         this._vimState.settings.scroll = vscode.workspace.getConfiguration("vim").get("scroll", 20) || 20;
+        this._vimState.settings.ignorecase = vscode.workspace.getConfiguration("vim").get("ignorecase", false) || false;
```

Comment:
Can we default smartcase to true? Even though Vim doesn't default it to true, I think we should, because it's better - in the same way that we show incremental search results, even though Vim doesn't by default.